sonic-thermalctld/scripts/thermalctld

```diff
@@ -618,6 +618,7 @@ class ThermalControlDaemon(daemon_base.DaemonBase):
         """
         super(ThermalControlDaemon, self).__init__(log_identifier)
         self.stop_event = threading.Event()
+        self.exit_code = -1
```

Comment:
Exit codes in Linux are non-negative values. `-1` will get converted to `255`. I suggest using a positive value here between 1 and 127 so that it's easy to search for in the code.